kafka/client.go

```diff
@@ -635,6 +637,15 @@ func (c *Client) topicConfig(topic string) (map[string]*string, error) {
 				log.Printf("[TRACE] Syonyms: %v", s)
 			}
 
+			if tConf.Name == "segment.bytes" {
+				re := regexp.MustCompile(`(?i)kafka-serverless\.(.*)\.amazonaws\.com`)
```

Comment:
we do something similar in https://github.com/Mongey/terraform-provider-kafka/blob/76b7c1da3920931bc330b6c9ee30ce93e07ba8ee/kafka/topic.go#L52 
can we unify these ?

Reply:
Can you elaborate on what you mean by unify? I think the `cleanup.policy` is uniquely different since it is settable at topic creation. I can certainly align them both to use either `for` or `slices`, whichever you prefer. My editor seemed to indicate slices is a newly preferred structure. Golang is something I touch only once in a long while so I'm open to suggestion.

Reply:
Sure, I just meant having a single function for figuring out if we're working with MSK serverless, so that we don't need to maintain the regex in two different files.

see  86e0512a03fea6fe1ae44f25130cd18aedf68029 and 74a13eb56f9a5dc7711bca409d36082af3cb60d3

Reply:
🤦‍♀️ of course. Thank you.